projects/core/components/hosted-dropdown/hosted-dropdown.component.ts

```diff
@@ -94,6 +96,11 @@ export class TuiHostedDropdownComponent implements TuiFocusableElementAccessor {
         @Inject(TuiDropdownHoverDirective)
         private readonly hover$: Observable<boolean> | null,
         @Inject(ElementRef) private readonly elementRef: ElementRef,
+        @Optional()
+        @Self()
+        @Inject(TuiDropdownHoverDirective)
+        readonly dropdownHover: TuiDropdownHoverDirective | null,
+        @Inject(TUI_IS_CYPRESS) readonly isCypress: boolean,
```

Comment:
I don't think components should know about cypress

Reply:
fixed